gen/ad_user_mu_ucn

```diff
@@ -113,6 +140,10 @@ for my $login (@logins) {
 	if (defined $mail and length $mail) {
 		print FILE "mail: " . $mail . "\n";
 	}
+	foreach (@{$users->{$login}->{$A_CHIPNUMBERS}}){
```

Comment:
I'm not sure, but I think this foreach with re-typing to array will fail if chip numbers attribute is `undef`. It can be, since I don't see any check (attribute module) in perun core. I think you should add check above like:
```perl
my $chipNumbers = $users->{$login}->{$A_CHIPNUMBERS};
if (defined $chipNumbers) {
    # ... LDIF print code ...
}
```